scilpy/tractograms/tests/test_streamline_and_mask_operations.py

```diff
@@ -0,0 +1,241 @@
+# -*- coding: utf-8 -*-
+
+import os
+import tempfile
+
+import nibabel as nib
+import numpy as np
+
+from dipy.io.streamline import load_tractogram
+
+from scilpy.io.fetcher import get_testing_files_dict, fetch_data, get_home
+from scilpy.image.utils import split_mask_blobs_kmeans
+from scilpy.tractograms.streamline_and_mask_operations import (
+    compute_streamline_segment,
+    cut_between_masks_streamlines,
+    cut_outside_of_mask_streamlines,
+    get_endpoints_density_map,
+    get_head_tail_density_maps)
+from scilpy.tractograms.streamline_and_mask_operations import \
+    _intersects_two_rois
+from scilpy.tractograms.uncompress import uncompress
+
+fetch_data(get_testing_files_dict(), keys=['tractograms.zip'])
+tmp_dir = tempfile.TemporaryDirectory()
+
+
+def _setup_files():
+    """ Load streamlines and masks relevant to the tests here.
+    """
+    os.chdir(os.path.expanduser(tmp_dir.name))
+    in_ref = os.path.join(get_home(), 'tractograms',
+                          'streamline_and_mask_operations',
+                          'bundle_4_wm.nii.gz')
+
+    in_head_tail = os.path.join(get_home(), 'tractograms',
+                                'streamline_and_mask_operations',
+                                'bundle_4_head_tail.nii.gz')
+    in_head_tail_offset = os.path.join(get_home(), 'tractograms',
+                                       'streamline_and_mask_operations',
+                                       'bundle_4_head_tail_offset.nii.gz')
+    in_center = os.path.join(get_home(), 'tractograms',
+                             'streamline_and_mask_operations',
+                             'bundle_4_center.nii.gz')
+    in_sft = os.path.join(get_home(), 'tractograms',
+                          'streamline_and_mask_operations',
+                          'bundle_4.tck')
+
+    # Load reference and roi files
+    reference = nib.load(in_ref)
+    head_tail_rois = nib.load(in_head_tail).get_fdata()
+    head_tail_offset_rois = nib.load(in_head_tail_offset).get_fdata()
+    center_roi = nib.load(in_center).get_fdata()
+
+    # Load sft
+    sft = load_tractogram(in_sft, reference)
+    return sft, reference, head_tail_rois, head_tail_offset_rois, center_roi
+
+
+def test_get_endpoints_density_map():
+    """ Test the get_endpoints_density_map function.
+    """
+
+    sft, reference, _, _, _ = \
+        _setup_files()
```

Comment:
Looks like it could fit on the previous line, I think. In other functions too.

Reply:
Yes, I "starred" the unused returned objects also.